src/main/java/slimeknights/tconstruct/TConstruct.java

```diff
@@ -225,27 +229,30 @@ private static void missingMappings(MissingMappingsEvent event) {
 
   /**
    * Gets a resource location for Tinkers
-   * @param name  Resource path
-   * @return  Location for tinkers
+   *
+   * @param name Resource path
+   * @return Location for tinkers
```

Comment:
Don't make arbitrary reformatting changes. They just clutter what needs to be reviewed.

Reply:
These formatting changes appear to have been all created by the IntelliJ formatter (or Eclipse formatter, but I use IDEA CE and it is making the exact same suggestions), including the `<p>` addition further down below in this file.  I pushed a commit to undo them for the time being as it does not directly relate to 1.20 porting, but it would probably be good to run a formatter in the future!

Reply:
That formatter has not been used in years.